src/flows/shared/steps/OrganizeGallery/CollectionRowSettings.tsx

```diff
@@ -21,9 +23,12 @@ type Props = {
 function CollectionRowSettings({ collection, wizard: { push } }: Props & WizardComponentProps) {
   const { showModal } = useModal();
   const { setCollectionIdBeingEdited } = useCollectionWizardActions();
+  const username = useAuthenticatedUsername();
 
   const { id, name, collectors_note, hidden } = collection;
 
+  const collectionUrl = `${window.location.origin}/${username}/${id}`;
```

Comment:
similar comment as above. looks like this article has some suggestions: https://stackoverflow.com/questions/65199051/how-to-get-page-url-or-hostname-in-nextjs-project

Reply:
Thanks for the link! Let me try this first